lib/FindLocation/FindLocationLookup.js

```diff
@@ -135,13 +134,17 @@ export const FindLocationLookup = (props) => {
     libraries,
   ]);
 
-  const renderBeforeSearch = useCallback(() => {
+  const renderBeforeSearch = useCallback((resetFilters) => {
     return crossTenant
       ? (
         <AffiliationsSelection
           affiliations={affiliations}
           label={affiliationsLabel}
-          onChange={onTenantChange}
+          onChange={(tenant) => {
+            onTenantChange?.(tenant);
+            resetFilters?.();
+            setFilters({});
```

Comment:
Probably also makes sense to create a const like `INIT_FILTERS` and use it on both places: in useState and when resetting to initials here

Reply:
sure